swatch-contracts/src/main/resources/application.properties

```diff
@@ -387,6 +437,3 @@ rhsm-subscriptions.subscription-sync-enabled=${SUBSCRIPTION_SYNC_ENABLED:true}
 
 # Enable Kafka metrics
 smallrye.messaging.observation.enabled=true
-
-# Disable Keycloak DEV services
-quarkus.keycloak.devservices.enabled=false
```

Comment:
why did you delete this property?

Reply:
It was a bad merged.  Now fixed.  Thank you.